build.gradle

```diff
@@ -42,7 +42,6 @@ dependencies {
     compile 'in.srain.cube:grid-view-with-header-footer:1.0.12'
     compile 'com.android.support:support-v4:23.0.1'
     compile 'com.android.support:gridlayout-v7:23.0.1'
-    debugCompile 'com.facebook.stetho:stetho:1.1.1'
```

Comment:
I took this out because we were needing to multidex. Without it we don't need to multidex... for now